apps/middleman/src/app/app/(sidebar)/(lists)/transactions/table/index.tsx

```diff
@@ -4,94 +4,95 @@ import { columns, filters, sorts } from './columns'
 import { DetailItem, Operation, useDetailContext } from '@/app/detail/Detail'
 import { MessageType } from '@/lib/constants'
 import { useQuery } from '@tanstack/react-query'
-import { GetUserTransactions } from '@/actions/Transactions'
+import {GetTransactions, GetUserTransactions} from '@/actions/Transactions'
 import { useEffect } from 'react'
 
 export default function TransactionsTable() {
-  const { data, isLoading, isError, refetch } = useQuery({
-    queryKey: ["transactions"],
-    queryFn: GetUserTransactions,
-    refetchInterval: 60000,
-  });
-  const {items, updateItem} = useDetailContext()
+    const { data, isLoading, isError, refetch } = useQuery({
+        queryKey: ["transactions"],
+        queryFn: GetTransactions,
```

Comment:
If I'm not mistaken, here you are calling this function:

```typescript
export async function GetTransactions() {
    const isAdmin = await isUserAdmin();

    if (!isAdmin) {
        throw new Error("Unauthorized");
    }

    return getTransactions();
}
```

It checks if the user is an admin, if not throws an error. This table is for the not admin users (customers?), so we need to keep using `GetUserTransactions`. Also change the `queryKey` for `userTransactions` or something like that.